cfsan-snp-pipeline/2.2.1/Dockerfile

```diff
@@ -0,0 +1,121 @@
+FROM ubuntu:xenial as app
+
+# metadata
+LABEL base.image="ubuntu:xenial"
+LABEL dockerfile.version="1"
+LABEL software="cfsan_snp_pipeline"
+LABEL software.version="2.2.1"
+LABEL description="A pipeline for the production of SNP matrices from sequence data used in the phylogenetic analysis of pathogenic organisms sequenced from samples of interest to food safety."
+LABEL website="https://github.com/CFSAN-Biostatistics/snp-pipeline"
+LABEL license="https://github.com/CFSAN-Biostatistics/snp-pipeline/blob/master/LICENSE.txt"
+LABEL original.maintainer="Justin Payne"
+LABEL original.maintainer.email="justin.payne@fda.hhs.gov"
+LABEL maintainer="Curtis Kapsak"
+LABEL maintainer.email="pjx8@cdc.gov"
+
+# This Dockerfile is based on a Docker file from Justin Payne which can be found here:
+# https://github.com/CFSAN-Biostatistics/snp-pipeline/blob/master/Dockerfile
+# The dockerfile has been modified from it's original form by Curtis Kapsak and Sam Baird
+
+WORKDIR /tmp/
+RUN apt-get update \
+	&& apt-get install -y --no-install-recommends \
+		build-essential \
+		bzip2 \
+		default-jre \
+		g++ \
+		gcc \
+		git \
+		gsl-bin \
+		libgsl0-dev \
+		libtbb-dev \
+		libbz2-dev \
+		liblzma-dev \
+		make \
+		ncurses-dev \
+		python \
+		python-dev \
+		tabix \
+		wget \
+		zlib1g-dev \
+        ca-certificates \
+	&& apt-get clean
+
+# Dependency version, can be updated in the build with build_args
+ARG SNP_PIPELINE_VER
+ENV SNP_PIPELINE_VER=${SNP_PIPELINE_VER:-2.2.1}
+
+ARG BCFTOOLS_VER
+ENV BCFTOOLS_VER=${BCFTOOLS_VER:-1.8}
+ARG BOWTIE2_VER
+ENV BOWTIE2_VER=${BOWTIE2_VER:-2.3.4.1}
+ARG HTSLIB_VER
+ENV HTSLIB_VER=${HTSLIB_VER:-1.3.2}
+ARG GATK_VER
+ENV GATK_VER=${GATK_VER:-3.4-46-gbc02625}
+ARG PICARD_VER
+ENV PICARD_VER=${PICARD_VER:-2.18.4}
+ARG SAMTOOLS_VER
+ENV SAMTOOLS_VER=${SAMTOOLS_VER:-1.8}
+ARG SRATOOLKIT_VER
+ENV SRATOOLKIT_VER=${SRATOOLKIT_VER:-2.8.1}
+ARG VARSCAN_VER
+ENV VARSCAN_VER=${VARSCAN_VER:-2.3.9}
+ARG NUMPY_VER
+ENV NUMPY_VER=${NUMPY_VER:-1.16.6}
+ARG BIOPYTHON_VER
+ENV BIOPYTHON_VER=${BIOPYTHON_VER:-1.68}
```

Comment:
I am unclear why these would need to be environmental variables. They perform some function during run time?

I would simplify this to 
```
ARG SNP_PIPELINE_VER=2.2.1
ARG BCFTOOLS_VER=1.8
ARG BOWTIE2_VER=2.3.4.1
ARG HTSLIB_VER=1.3.2
ARG GATK_VER=3.4-46-gbc02625
ARG PICARD_VER=2.18.4
ARG SAMTOOLS_VER=1.8
ARG SRATOOLKIT_VER:-2.8.1
ARG VARSCAN_VER=2.3.9
ARG NUMPY_VER=1.16.6
ARG BIOPYTHON_VER=1.68
```